### ISSUE TITLE
CMD+Enter does not work when editing a message

### ISSUE THREAD

**Issue by ftvkyo** [mattermost/mattermost-server#14934]
#### Summary
When CMD+Enter shortcut for sending messages is enabled, it does not work in the message editing popup window.

#### Steps to reproduce
Mattermost Team Edition, Server 5.24.1
1. Enable CMD+Enter shortcut for message sending.
2. Send a message anywhere.
3. Start editing the message.
4. Try to submit your edits pressing CMD+Enter.

#### Expected behavior
Edits should be submitted, editing popup window should be closed.

#### Observed behavior (that appears unintentional) 
Nothing happens on CMD+Enter press.

#### Possible fixes
Probably

https://github.com/mattermost/mattermost-webapp/blob/2d0a531ec28dd5a909a6df30be3dab9496b7544e/components/edit_post_modal/edit_post_modal.jsx#L292

should be changed to
```
        } else if ((this.props.ctrlSend || this.props.codeBlockOnCtrlEnter) && Utils.isKeyPressed(e, KeyCodes.ENTER) && e.ctrlKey || e.metaKey) {
```

Just like it's done here:

https://github.com/mattermost/mattermost-webapp/blob/2d0a531ec28dd5a909a6df30be3dab9496b7544e/components/create_post/create_post.jsx#L1025

I am not 100% sure though because you also have some logic for CMD+Enter in code blocks.

**Comment by amyblais** [comment]
@ftvkyo This sounds like something that we haven't implement before but is a good idea. Would you be open to sharing this on our feature request forum here https://mattermost.uservoice.com/forums/306457-general?

**Comment by ftvkyo** [comment]
Ah, I just thought that the behaviour of the editing window and behaviour of the message box should be the same in this meaning -- as they actually have the same editing functions and edit the same text.

The problem is that without that fix people with CMD+Enter option enabled have to click the save button with their mouse.

Sure, thank you, I will make a feature request then 🙃

**Comment by amyblais** [comment]
@ftvkyo It appears this is actually a bug and a ticket has been created here: https://mattermost.atlassian.net/browse/MM-26700. Apologies for having you create a feature request for this earlier.

**Comment by ftvkyo** [comment]
@amyblais thanks, that's great.

I should note that the step 3 about
"making a code block" in the link you sent is not necessary, but that's probably not a problem.

**Comment by amyblais** [comment]
Closing this as this was fixed for v5.31. QA tested and was able to post edited messages using CTRL+Enter, ⌘Enter and even CTRL⌘+Enter. Using just ‘Enter’ just added a new line as expected.

Please re-open this issue if you still experience this problem in the latest server version,
